qaframework-bdd-tests/src/test/resources/features/refapp-2.x/13-appointment/appointment.feature

```diff
@@ -0,0 +1,66 @@
+Feature: Appointment Management
+
+  Background:
+    Given a user clicks on Request Appointment link from the Patient dashboard
+    When the system loads Request Appointment page
+    And a user fills in the appointment request details
+    And a user clicks save button
+    Then the system adds appointment request into the appointment requests table
+
+  @selenium
+  @appointment
+  Scenario: Delete Appointment Request
+    When a user clicks on Schedule Appointment link from the Patient dashboard
+    Then the system loads Manage Appointments page
+    And a user clicks on Cancel this Request icon
+    Then the system deletes appointment request from the request table
+
+  @selenium
+  @appointment
+  Scenario: Create, Edit and Delete Appointment Block
+    #User Story: Create Appointment Block
+    When a user clicks Appointment scheduling app from the home page
+    Then the system loads the appointment scheduling page
+    And a user clicks on Manage Provider Schedules app
+    And the system loads Manage Provider Schedules page
+    And a user fills in the details of the appointment block
```

Comment:
@Irenyak1  This line is breaking the ci with these logs https://ci.openmrs.org/browse/CONTRIB-QA-FRAMEWORK-1403/log  . Cross check with the method named `[clickOnNextWeekDay](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageProviderSchedulesPage.java#L20)`  and see the elements used whether they meet the standards. Then update it with your feature files.

Reply:
@sherrif10 Let me look into it.